jms/1.1/modules/ROOT/pages/jms-connector-reference.adoc

```diff
@@ -586,12 +586,8 @@ The maximum number of times a message can be redelivered and processed unsuccess
 
 Used to configure the RedeliveryPolicy#getMaximumRedeliveries().
  0 means no redelivery. -1 means infinite re deliveries are accepted. | 0 |
-| Trusted Packages a| Array of String | White list of packages of classes that are allowed sent and received.
- This property starting versions 5.12.2 and 5.13.0, ActiveMQ enforces users to explicitly white list packages that
- can be exchanged using ObjectMessages. |  |
-| Trust All Packages a| Boolean | Indicates if any class from any package can be sent and received. Enabling this is unsafe because a malicious payload can exploit the host system.
- This property starting versions 5.12.2 and 5.13.0, ActiveMQ enforces users to explicitly white list packages that
- can be exchanged using ObjectMessages. | false |
+| Trusted Packages a| Array of String | Allowed packages of classes to send and receive. For this parameter, ActiveMQ starting versions 5.12.2 and 5.13.0 enforce users to allow packages that can be exchanged using ObjectMessages. |  |
```

Comment:
Some ungrammatical text. I suggest something like this but I am not sure if I am interpreting the current text correctly:

... Starting with versions 5.12.2 and 5.13.0, ActiveMQ requires that you set either this parameter or the Trust All Packages parameter to allow packages that can be exchanged using ObjectMessages.